--- pam/meson.build	2026-07-27 15:56:43.016726720 -0400
+++ patches/meson.build.new	2026-07-26 19:42:29.580534811 -0400
@@ -3,9 +3,7 @@
   'c',
   version : '1.7.2',
   license : 'BSD-3-Clause OR GPL-2.0-or-later',
-  default_options : ['b_pie=true',
-    'prefix=/usr',
-    'warning_level=2'],
+  default_options : ['b_pie=true', 'prefix=/usr', 'warning_level=2'],
   meson_version : '>= 0.62.0'
 )
 
@@ -31,7 +29,8 @@
 endif
 
 
-cc = meson.get_compiler('c')
+# cc = meson.get_compiler('c')
+cc = meson.get_compiler('gcc')
 cdata = configuration_data()
 
 null_dep = dependency('', required: false)
@@ -123,7 +122,6 @@
 try_cc_flags = [
   '-Wbad-function-cast',
   '-Wcast-align',
-  '-Wno-discarded-qualifiers',
   '-Wcast-align=strict',
   '-Wcast-qual',
   '-Wdeprecated',
@@ -645,17 +643,13 @@
 libpam_inc = include_directories('libpam' / 'include')
 
 
-subdir('po')
 subdir('libpam_internal')
 subdir('libpam')
-subdir('libpamc')
 subdir('libpam_misc')
 if enable_docs
   subdir('doc')
 endif
-subdir('tests')
 subdir('modules')
-subdir('conf' / 'pam_conv1')
 if get_option('examples')
   subdir('examples')
 endif
